README.md

```diff
@@ -18,75 +17,95 @@ npm start
+      ...
+    }
+
+    p5.draw = () => {
+      ...
+    };
+  }
 
-An Sketch could look like this:
+  return <P5Wrapper sketch={sketch} />;
+}
 
+export default App;
 ```
-export default function sketch (p) {
-  let rotation = 0;
 
-  p.setup = function () {
-    p.createCanvas(600, 400, p.WEBGL);
-  };
+### Properties
 
-  p.myCustomRedrawAccordingToNewPropsHandler = function (props) {
-    if (props.rotation !== null){
-      rotation = props.rotation * Math.PI / 180;
-    }
-  };
-
-  p.draw = function () {
-    p.background(100);
-    p.normalMaterial();
-    p.noStroke();
-    p.push();
-    p.rotateY(rotation);
-    p.box(100);
-    p.pop();
-  };
-};
-```
+- `sketch`: This is the sketch script which should be executed in the p5 canvas.
+- You can also add as many custom properties as you want.
 
-In the Example above you see the `myCustomRedrawAccordingToNewPropsHandler` function.
-This function is called if Properties of the wrapper component are changing.
-In this case the Wrapper Component would be integrated like this: `<P5Wrapper sketch={sketch} rotation={rotation}/>`.
+In the below example you see the `myCustomRedrawAccordingToNewPropsHandler` function, which is called when the properties of a wrapper component are changed.
 
-### Properties
+```js
+// file:src/App.jsx
+import P5Wrapper from 'react-p5-wrapper';
 
-* sketch: This is the Sketch Script which should be executed in the P5 Canvas
-* You can add as many custom Properties as you want
+function App() {
+  let sketch = (p5) => {
+    let rotation = 0;
+
+    p5.setup = () => p5.createCanvas(600, 400, p5.WEBGL);
+
+    p5.myCustomRedrawAccordingToNewPropsHandler = (props) => {
+      if (props.rotation) {
+        rotation = (props.rotation * Math.PI) / 180;
+      }
+    };
+
+    p5.draw = () => {
+      p5.background(100);
+      p5.normalMaterial();
+      p5.noStroke();
+      p5.push();
+      p5.rotateY(rotation);
+      p5.box(100);
+      p5.pop();
+    };
+  }
+
+  return <P5Wrapper sketch={sketch} rotation={rotation}/>;
+}
+
+export default App;
+```
 
 ### Children
-* To Render a component on top of the sketch, simply add it as a child of the P5Wrapper component
 
+To render a component on top of the sketch, simply add it as a child of the `P5Wrapper` component.
 
-## Development (`src`, `lib` and the build process)
+## Development
 
-**NOTE:** The source code for the component is in `src`.
+**NOTE:** The source code for the component is in `src` directory.
 
-To build, watch and serve the examples (which will also watch the component source), run `npm start`.
+To build, watch and serve the examples which will also watch the component source, run:
+
+```
+npm start
+```
 
 ## License
 
 The MIT License (MIT)
 
-Copyright (c) 2016 Andreas Wolf and [contributors](https://github.com/and-who/react-p5-wrapper/graphs/contributors).
+Copyright (c) 2016 - 2021 Andreas Wolf and [contributors](https://github.com/and-who/react-p5-wrapper/graphs/contributors).
```

Comment:
This change can be removed as licences don't need kept up to date actually.

Reply:
Good point and thank you!